test/scenario/metering-plans/src/test/test.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const oauth = require('abacus-oauth');
+const request = require('abacus-request');
+const util = require('util');
+const uuid = require('uuid');
+
+const doGet = util.promisify(request.get);
+const doPost = util.promisify(request.post);
+
+const env = {
```

Comment:
I would name this variable `cfg`, as some of the values are defaults. Furthermore, someone might get confused further down the code that we are reading environment variables there.